### ISSUE TITLE
Issue with command line "ontogpt --help"

### ISSUE THREAD

**Issue by Gaurang111** [monarch-initiative/ontogpt#246]
I am trying to work with Ontogpt, been following the Quick Start guide to get started, but I am getting TypeError when running command line `ontogpt --help.`

(venv) PS C:\Users\goran\PycharmProjects\pythonProject> ontogpt --help   
INFO:root:API KEY path = C:\Users\goran\AppData\Local\ontology-access-kit\ontology-access-kit\ncbi-email-apikey.txt
INFO:root:API KEY path = C:\Users\goran\AppData\Local\ontology-access-kit\ontology-access-kit\ncbi-key-apikey.txt
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Scripts\ontogpt.exe\__main__.py", line 4, in <module>
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Lib\site-packages\ontogpt\cli.py", line 30, in <module>
    from ontogpt.engines import create_engine
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Lib\site-packages\ontogpt\engines\__init__.py", line 1, in <module>
    from ontogpt.engines.resolver import create_engine  # noqa
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Lib\site-packages\ontogpt\engines\resolver.py", line 6, in <module>
    from ontogpt.engines.enrichment import EnrichmentEngine
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Lib\site-packages\ontogpt\engines\enrichment.py", line 15, in <module>
    from ontogpt.engines.knowledge_engine import KnowledgeEngine
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Lib\site-packages\ontogpt\engines\knowledge_engine.py", line 28, in <module>
    from ontogpt.templates.core import ExtractionResult, NamedEntity
  File "C:\Users\goran\PycharmProjects\pythonProject\ontogpt\venv\Lib\site-packages\ontogpt\templates\core.py", line 16, in <module>
    class ConfiguredBaseModel(BaseModel,
  File "pydantic\main.py", line 282, in pydantic.main.ModelMetaclass.__new__
  File "<frozen abc>", line 106, in __new__
TypeError: ConfiguredBaseModel.__init_subclass__() takes no keyword arguments

I am not sure where is the problem, I've built a new Virtual Environment, upgraded all the packages, and generated a new openai APi key. but can't get rid of this error. Am I supposed to make any changes in `ConfiguredBaseModel`?

**Comment by plbremer** [comment]
Maybe this is a fresh version issue, because i just downloaded and am getting the same problem 

`(venv_ontogpt) pgleasonbremer@MacBook-Pro-4 ontogpt % ontogpt --help
INFO:root:API KEY path = /Users/pgleasonbremer/Library/Application Support/ontology-access-kit/ncbi-email-apikey.txt
INFO:root:Email for NCBI API not found.
INFO:root:API KEY path = /Users/pgleasonbremer/Library/Application Support/ontology-access-kit/ncbi-key-apikey.txt
INFO:root:NCBI API key not found. Will use no key.
Traceback (most recent call last):
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/bin/ontogpt", line 5, in <module>
    from ontogpt.cli import main
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/lib/python3.10/site-packages/ontogpt/cli.py", line 30, in <module>
    from ontogpt.engines import create_engine
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/lib/python3.10/site-packages/ontogpt/engines/__init__.py", line 1, in <module>
    from ontogpt.engines.resolver import create_engine  # noqa
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/lib/python3.10/site-packages/ontogpt/engines/resolver.py", line 6, in <module>
    from ontogpt.engines.enrichment import EnrichmentEngine
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/lib/python3.10/site-packages/ontogpt/engines/enrichment.py", line 15, in <module>
    from ontogpt.engines.knowledge_engine import KnowledgeEngine
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/lib/python3.10/site-packages/ontogpt/engines/knowledge_engine.py", line 28, in <module>
    from ontogpt.templates.core import ExtractionResult, NamedEntity
  File "/Users/pgleasonbremer/coding_projects/altos/ontogpt/venv_ontogpt/lib/python3.10/site-packages/ontogpt/templates/core.py", line 16, in <module>
    class ConfiguredBaseModel(BaseModel,
  File "pydantic/main.py", line 282, in pydantic.main.ModelMetaclass.__new__
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/abc.py", line 106, in __new__
    cls = super().__new__(mcls, name, bases, namespace, **kwargs)
TypeError: ConfiguredBaseModel.__init_subclass__() takes no keyword arguments`

**Comment by caufieldjh** [comment]
Hi @Gaurang111 and @plbremer, thanks for the issue.
It looks like this is happening due to the wrong version of `pydantic` getting installed due to an upstream dependency change (in `prefixmaps`, specifically).

As a workaround, try to install a version of `pydantic` of v2 or above:
```
pip install "pydantic>2.0"
```

You'll likely get an error about `prefixmaps` not being compatible but it should alleviate that TypeError.

**Comment by Vishal-Joshi** [comment]
I have tried the resolution but it still does not work

I get following error after I have done `pip install "pydantic>2.0"`

`Traceback (most recent call last):
  File "/homes/vjoshi/.conda/envs/ontogpt/bin/ontogpt", line 5, in <module>
    from ontogpt.cli import main
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/ontogpt/cli.py", line 34, in <module>
    from ontogpt.engines.gpt4all_engine import GPT4AllEngine  # type: ignore
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/ontogpt/engines/gpt4all_engine.py", line 34, in <module>
    from ontogpt.utils.gpt4all_runner import chain_gpt4all_model, set_up_gpt4all_model
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/ontogpt/utils/gpt4all_runner.py", line 5, in <module>
    import llm
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/llm/__init__.py", line 15, in <module>
    from .plugins import pm
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/llm/plugins.py", line 13, in <module>
    pm.load_setuptools_entrypoints("llm")
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/pluggy/_manager.py", line 398, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/importlib_metadata/__init__.py", line 209, in load
    module = import_module(match.group('module'))
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/llm_gpt4all.py", line 1, in <module>
    from gpt4all import GPT4All as _GPT4All
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/gpt4all/__init__.py", line 1, in <module>
    from .gpt4all import Embed4All, GPT4All  # noqa
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/gpt4all/gpt4all.py", line 18, in <module>
    from . import pyllmodel
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/gpt4all/pyllmodel.py", line 42, in <module>
    llmodel = load_llmodel_library()
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/gpt4all/pyllmodel.py", line 32, in load_llmodel_library
    lib = ctypes.CDLL(str(MODEL_LIB_PATH / f"libllmodel.{ext}"))
  File "/homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/ctypes/__init__.py", line 382, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /homes/vjoshi/.conda/envs/ontogpt/lib/python3.9/site-packages/gpt4all/llmodel_DO_NOT_MODIFY/build/libllmodel.so)`

**Comment by serenalotreck** [comment]
@Vishal-Joshi there is a fix for this! It's discussed in #221 

However, @caufieldjh, when we closed that issue I had only tried `ontogpt --help`, which does run after changing the version of `gpt4all`, but now that I'm trying to run the toy example in the setup section of the docs, I'm getting a `PermissionError`. Going to open a new issue since I assume it's not related but figured I'd mention it here @Vishal-Joshi since #221 may not solve the issue entirely!
